### DISCUSSION TITLE
Issa Chat App - A simple way to chat with ChatGPT on Android

### DISCUSSION THREAD

**Discussion by BrianJr03** [CJCrafter/ChatGPT-Java-API#16] [Show and tell]
Hola, over the past few weeks, I've developed a simple way to chat with ChatGPT on Android. The app is called `Issa Chat App`

I'm currently using the now archived [ChatGPT-Kotlin-API](https://github.com/CJCrafter/ChatGPT-Kotlin-API) but will switch to this API soon.

You can try the app here: [Latest release](https://github.com/BrianJr03/Issa-Chat-App/releases)

[![Github All Releases](https://img.shields.io/github/downloads/BrianJr03/Issa-AI-App/total.svg)]() 

## Features
- ### Chat with chatGPT
  - Only an OpenAI API Key is needed to chat with chatGPT (`gpt-3.5-turbo`)
  - Add `Conversational Context` to personalize responses from chatGPT
    - Example: "You are my sarcastic assistant"

- ### Chat taps

  - Single tap to `toggle` a Chat's date and time

  - Double tap to `play` a Chat's text as audio
  
  - Long tap to `copy` a Chat's text
  
- ### Dark / Light mode
  - Based on your system settings, this app supports both dark and light mode
  
- ### Select app theme
  - Select between 3 themes in Theme dialog  
  
- ### Persistent Chat
  - Chats will be saved automatically
  
- ### Settings Dialog
  - You can do a few things in `Settings`
    - Add & Update API Key
    - Remove all chats
    - Toggle `Auto-play`, which allows one to automatically play an incoming Chat's text
    - Toggle a random `Conversational Context` being set on app start

**Comment by CJCrafter** [comment]
That GUI is clean. I recommend checking out the `stream` option for a "live preview" of what is being generated. This should make your APP more user friendly (So they don't have to wait so long for their message).

**Reply by BrianJr03** [reply]
I'll def check that out. Thanks
